src/coreclr/src/jit/lclvars.cpp

```diff
@@ -6567,8 +6574,8 @@ void Compiler::lvaAssignVirtualFrameOffsetsToLocals()
     {
 #if defined(TARGET_AMD64) && !defined(UNIX_AMD64_ABI) // No 4 slots for outgoing params on System V.
         noway_assert(lvaOutgoingArgSpaceSize >= (4 * TARGET_POINTER_SIZE));
-#endif
         noway_assert((lvaOutgoingArgSpaceSize % TARGET_POINTER_SIZE) == 0);
+#endif
```

Comment:
Doesn't this eliminate the `noway_assert` for platforms it would still be valid?

Reply:
You are right and actually, it should be true even for arm64 apple, because they alight outgoing arg space size as well.
Fixed.